src/olympia/activity/tests/test_utils.py

```diff
@@ -274,6 +274,29 @@ def test_reviewer_reply(self, send_mail_mock):
         self._check_email(send_mail_mock.call_args_list[1],
                           self.addon.get_dev_url('versions'))
 
+    @mock.patch('olympia.activity.utils.send_mail')
+    def test_log_with_no_comment(self, send_mail_mock):
+        # One from the reviewer.
+        self._create(amo.LOG.REJECT_VERSION, self.reviewer)
+        action = amo.LOG.APPROVAL_NOTES_CHANGED
+        comments = None
+        log_and_notify(action, comments, self.developer, self.version)
```

Comment:
I feel like you should directly pass `None` to make it more obvious what get's passed.

Generally, here and for many other `log_and_notify` calls I'm a fan of passing ``kwargs`` instead of just ``args``, e.g ``log_and_notify(action=..., comments=...)`` with that many arguments it get's easier to read and reason about.